grpc4bmi/bmi_julia_model.py

```diff
@@ -0,0 +1,655 @@
+from typing import List
+
+from bmipy import Bmi
+import numpy as np
+from juliacall import Main as jl, ModuleValue, TypeValue
+
+def install(package):
+    """Add package to Julia environment.
+    
+    Args:
+        package: Name of package to install.
+    """
+    jl.Pkg.add(package)
```

Comment:
Specific URLs and git branches can also be installed. See https://pkgdocs.julialang.org/v1/api/#Package-API-Reference

```julia
import Pkg
Pkg.add(name="Wflow", rev="BMI")
```
or

```julia
Pkg.add(url="https://github.com/JuliaLang/Example.jl", rev="master")
```

Reply:
Removed install method and use `jl.Pkg.add()` direct;ly.